.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+charts/.DS_Store
```

Comment:
This line isn't necessary and can be removed.

Reply:
Thanks for the heads-up, I removed it.

Reply:
Clarification, just remove `charts/.DS_Store` since `.DS_Store` is already present.

Reply:
Yep, I removed `charts/.DS_Store` it just didn't show it here. The change is here: https://github.com/slateci/slate-catalog-incubator/pull/48/commits/0bc57477b0e81ac07734402c825d5bdb053a8f33